### PR TITLE
data:AlokMahapatra26

### PR DESCRIPTION
**Pull request by AlokMahapatra26**

### CODE REVIEW

File: data/AlokMahapatra26.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{
+  "name": "Alok Mahapatra",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Tech Enthusiast | Web Developer | UIUX ",
+  "avatar": "https://github.com/AlokMahapatra26.png",
+  "tags": [
+    "Javascript",
+    "UI UX",
+    "MERN Stack",
```
Comment (Aadarsh805) [inline data/AlokMahapatra26.json:12]: remove this comma at the end of line 12 after `Mern Stack`

File: data/AlokMahapatra26.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    {
+      "name": "GitHub",
+      "url": "https://github.com/AlokMahapatra26",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/alok-mahapatra/",
+      "icon": "FaLinkedin"
+    },
```
Comment (Aadarsh805) [inline data/AlokMahapatra26.json:44]: remove this comma too at the end of line 44

File: data/AlokMahapatra26.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   
+  ],
+  "milestones": [
+    
+    {
+      "title": "Started My tech Journey",
+      "date": "September 2022",
+      "icon": "FaRegCompass",
+      "color": "red"
+    },
```
Comment (Aadarsh805) [inline data/AlokMahapatra26.json:55]: remove this comma as well at the end of line 55

**Review by Pal-Sandeep — DISMISSED** [review]
@AlokMahapatra26 Looks good to me

File: data/AlokMahapatra26.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
+
+
```
Comment (Panquesito7) [inline data/AlokMahapatra26.json:2]: There are two extra lines here.
```suggestion
```

File: data/AlokMahapatra26.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  ],
+  
+  
+  "links": [
+   
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/aloktwts",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
```
Comment (Panquesito7) [inline data/AlokMahapatra26.json:25]: Hmm. Not sure you can use color fields in the links. 🤔

Comment (LinuxGamer) [reply data/AlokMahapatra26.json:25]: You can, idk the specifics, but colors can be set on links.

**Review by LinuxGamer — APPROVED** [review]
Looks all good to me :) 👍🏼

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**